src/server.ts

```diff
@@ -1,11 +1,14 @@
-import 'reflect-metadata';
 import { Container } from 'typedi';
 
-import { bootstrapContainers, bootstrapDB, BootstrapServer } from './utils';
-
-bootstrapContainers();
-
-const server = Container.get(BootstrapServer);
-server.startListening();
-
-bootstrapDB();
+import { App } from './libs/app';
+import { Database } from './libs/database';
+
+(async() => {
```

Comment:
Personally, I don't like having an IIFE in a typescript file as it belongs to `.js` file pattern. The codes inside the IIFE is still executed even without it.

Reply:
I see, we can omit this one